src/main/java/robot/OI.java

```diff
@@ -178,7 +179,27 @@ public OI() {
         TESSERACTED,
         SINE
     }
-
+    //A method which sets independent intensity for the left and right for pilot//
```

Comment:
fix spacing for all setters, change comments to javadocs, and move them to underneath the getters